lib/Dialect/FIRRTL/Transforms/ModuleInliner.cpp

```diff
@@ -655,9 +655,11 @@ Inliner::mapPortsToWires(StringRef prefix, OpBuilder &b,
 
     // Compute a unique symbol if needed
     StringAttr newSym;
-    StringAttr oldSym = portInfo[i].sym;
-    if (!oldSym.getValue().empty())
+    StringAttr oldSym;
```

Comment:
Similarly, there's a TODO here to handle generating new names for field symbols on inline ports, right?

Reply:
For all the passes we need to add support for per field symbols. Didn't want to tackle that in this PR. As that might require some update to logic for the different passes. This PR just assumes only `fieldID=0` symbol is relevant and ignores the rest. We need subsequent PRs to update all the passes to handle this.